Cargo.lock

```diff
@@ -19,9 +19,9 @@ checksum = "7079075b41f533b8c61d2a4d073c4676e1f8b249ff94a393b0595db304e0dd87"
 
 [[package]]
 name = "anyhow"
-version = "1.0.75"
+version = "1.0.76"
```

Comment:
Did you intentionally bump a bunch of these deps? I don't see how adding `fs2` would have changed this 🤔.